server/chainquery/index.js

```diff
@@ -0,0 +1,80 @@
+const Sequelize = require('sequelize');
+const logger = require('winston');
+
+import abnormalClaimTable from './tables/abnormalClaimTable';
+import addressTable from './tables/addressTable';
+import applicationStatusTable from './tables/applicationStatusTable';
```

Comment:
This table is used for tracking background job statuses internal to chainquery. Most likely not useful for speech.